test/components/selection-list.spec.tsx

```diff
@@ -159,25 +149,6 @@ describe('<SelectionList />', () => {
         expect(empty).to.have.height.at.most(full);
     });
 
-    it('Renders a divider', async () => {
```

Comment:
This test is not related to a theme, it should stay here.

Reply:
Divider will not have stylable className without passing theme. because it is speparate component now. it may be changed, but as approach I don't think it is good, it may be not rendered at all without theme.

Reply:
> Divider will not have stylable className without passing theme.

It will still have the root classname from src/component/.../divider.st.css, right? Why can't we check that one?

Reply:
it requires `divider.st.css` then